src/main/java/org/jenkinsci/plugins/casc/BaseConfigurator.java

```diff
@@ -78,7 +78,14 @@
             // See https://github.com/jenkinsci/structs-plugin/pull/18
             final Symbol s = setter.getAnnotation(Symbol.class);
             if (s != null) {
-                attribute.preferredName(s.value()[0]);
+                final String[] values = s.value();
+                final int last = values.length - 1;
+                // we assume preferred name is the last added as @Symbol value
```

Comment:
No, this directly contradicts `@Symbol` Javadoc:

> Multiple symbols can be specified as aliases.
> The first one is used as the primary identifier for reverse-mapping.

Reply:
Good point. But we have an issue with `DumbSlave`: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/slaves/DumbSlave.java#L68

created https://github.com/jenkinsci/jenkins/pull/3475